collectors/ebpf.plugin/README.md

```diff
@@ -8,14 +8,15 @@ sidebar_label: "eBPF"
 
 # eBPF monitoring with Netdata
 
-Netdata's extended Berkeley Packet Filter (eBPF) collector monitors kernel-level metrics for file descriptors, virtual
-filesystem IO, and process management on Linux systems. You can use our eBPF collector to analyze how and when a process
-accesses files, when it makes system calls, whether it leaks memory or creating zombie processes, and more.
+eBPF is a Linux kernel technology that allows us to run data collection programs within a kernel context.
```

Comment:
My suggestion is to start in a more generic definition like: "eBPF is a mechanism for Linux applications to execute code in Linux kernel space." and then make it more specific like: "eBPF has already been used to create programs for data collection, networking, debugging, tracing, firewalls, and more." And later on, we describe (as we already do) how Netdata is using it (Netdata has written many....)

Reply:
@cpipilas what is your opinion about to remove this first sentence completely and start with the nexŧ? I think we will achieve what you are suggesting with this action.

Reply:
Yes, it should be ok